### PR TITLE
windows: updates based on tests for gitlab

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Some updates for windows build based on the tests made to make it work in GitLab
need this PR to enable the build for windows

MR for the internal pipeline: https://git.internal.mattermost.com/mattermost/ci/focalboard/-/merge_requests/2

### CODE REVIEW

**Review by metanerd — APPROVED** [review]
:tada:

File: win-wpf/build.bat [hunk 1]
```diff
@@ -3,6 +3,9 @@
 WHERE msbuild.exe > nul 2>&1
 IF %ERRORLEVEL% NEQ 0 set PATH=%PATH%;C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin
 
+WHERE msbuild.exe > nul 2>&1
```
Comment (harshilsharma63) [inline win-wpf/build.bat:6]: Do we want to remove the above where block? They seem to be for the same condition

**Review by harshilsharma63 — CHANGES_REQUESTED** [review]
One question.

File: win-wpf/build.bat (hunk 1 above)
Comment (cpanato) [reply win-wpf/build.bat:6]: one if the Community which looks like is the one used in github actions and the other one is for Enterprise which we use internally

**Review by harshilsharma63 — APPROVED** [review]

**Comment by harshilsharma63** [comment]
LGTM!

**Comment by cpanato** [comment]
I will merge this, @chenilim if you have any comments we can do in a followup if needed

**Pull request merged**